docs/FAQ.md

```diff
@@ -11,11 +11,11 @@ Apache Spark caches what is used to build the output of the `explain()` function
 knowledge about configs, so it may return results that are not up to date with the current config
 settings. This is true of all configs in Spark. If you changed
 `spark.sql.autoBroadcastJoinThreshold` after running `explain()` on a `DataFrame`, the resulting
-query would not change to reflect that config and still show a `SortMergeJoin` even though the
-new config might have changed to be a `BroadcastHashJoin` instead. When actually running something
-like with `collect`, `show` or `write` a new `DataFrame` is constructed causing spark to re-plan the
-query. This is why `spark.rapids.sql.enabled` is still respected when running, even if explain
-shows stale results.
+query would not change to reflect that config and still show a `SortMergeJoin` even though the new
+config might have changed to be a `BroadcastHashJoin` instead. When actually running something like
+with `collect`, `show` or `write` a new `DataFrame` is constructed causing spark to re-plan the
```

Comment:
```suggestion
with `collect`, `show` or `write` a new `DataFrame` is constructed causing Spark to re-plan the
```

Reply:
The intent with the line length changes was to make line lengths 100 (similar to the scala style recommendation https://github.com/databricks/scala-style-guide#line-length ).

Reply:
Understood, but paragraphs that were not over the 100 character limit were also reformatted unnecessarily.  See the changes to the first paragraph in compatibility.md as one example.  If there's a reformatting tool you're using for this and everyone is supposed to use it, we should document it (and audit/automate it as much as possible) otherwise we're destined for another "let's rewrite the docs for formatting" PR after a while.